packages/blade/src/components/BottomSheet/_decisions/decisions.md

```diff
@@ -0,0 +1,225 @@
+| --------------- | ----------- | ----------------------- | ----------------------------------------------------------------------------------------------- | -------- |
+| open            | `boolean`   | `false`                 | toggles bottom sheet state content                                                              |          |
+| snapPoints      | `string[]`  | `['25%', '50%', '85%']` | snappoints in which the bottom sheeet will rest on                                              |          |
+| onDismiss       | `Callback`  | `undefined`             | called when the bottom sheet is closed, either by user state, hitting `esc` or tapping backdrop |          |
+| initialFocusRef | `React.Ref` | `undefined`             | ref element you want to get keyboard focus when opening the sheet                               |          |
+
+### `BottomSheetBody`
+
+| Prop       | Type              | Default     | Description                 | Required |
+| ---------- | ----------------- | ----------- | --------------------------- | -------- |
+| `children` | `React.ReactNode` | `undefined` | Contents of the BottomSheet | ✅       |
+
+### `BottomSheetHeaderLeading` API
+
+| Prop     | Type            | Default     | Description                          | Required |
+| -------- | --------------- | ----------- | ------------------------------------ | -------- |
+| `title`  | `string`        | `undefined` | Title of the Card                    | ✅       |
+| `prefix` | `IconComponent` | `undefined` | Prefix icon placed before title text |          |
+
+### `BottomSheetHeaderTrailing` API
+
+| Prop     | Type                 | Default     | Description                                                | Required |
+| -------- | -------------------- | ----------- | ---------------------------------------------------------- | -------- |
+| `visual` | `Link, Action, Icon` | `undefined` | Trailing visual element placed on right side of the header |          |
+
+### `BottomSheetFooterLeading` API
+
+| Prop     | Type            | Default     | Description                          | Required |
+| -------- | --------------- | ----------- | ------------------------------------ | -------- |
+| `title`  | `string`        | `undefined` | Title of the BottomSheet footer      | ✅       |
+| `prefix` | `IconComponent` | `undefined` | Prefix icon placed before title text |          |
+
+### `BottomSheetFooterTrailing` API
+
+| Prop      | Type                                     | Default     | Description                                | Required |
+| --------- | ---------------------------------------- | ----------- | ------------------------------------------ | -------- |
+| `actions` | `{ primary: Action, secondary: Action }` | `undefined` | Renders a primary/secondary action buttons |          |
+
+```ts
+type Action = {
+  onClick: () => void;
+  text: string;
+  type?: 'button' | 'reset' | 'submit';
```

Comment:
What would be purpose of reset and submit?

Reply:
Just accepting the same props as the Button. 

Tho might be useful if someone uses the bottomsheet as form submission modal.